src/components/LandingPage/data/smpData.tsx

```diff
@@ -15,4 +15,16 @@ export const SMPList: CardItem[] = [
     time: "5 min",
     link: "/tutorials/self-managed-enterprise-edition/use-an-external-mongodb-database",
   },
+  {
+    title: "Configure a self-managed PostgreSQL database",
+    module: MODULES.smp,
+    icon: "img/logo.svg",
+    description: (
+      <>Learn how to use an external PostgreSQL database with Self-Managed Enterprise Edition installations.</>
+    ),
+    newDoc: false,
```

Comment:
```suggestion
    newDoc: true,
```

Set to true to show the "new" flag on the tile. When you add another tutorial in the future, change this to false then.